_security/configuration/demo-configuration.md

```diff
@@ -27,14 +27,15 @@ The demo configuration is automatically called as part of the setup for each sup
 Use the following steps to set up the Security plugin using Docker:
 
 1. Download [docker-compose.yml](https://opensearch.org/downloads.html).
-2. In the `docker-compose.yml` file, set `DISABLE_SECURITY_PLUGIN` to `false`.
-3. Run the following command:
+2. Run the following command:
 
 ```bash
 docker-compose up
 ```
 {% include copy.html %}
 
+If you want to disable the Security plugin when using Docker, set the `DISABLE_SECURITY_PLUGIN` environment variable  to `true` in the `docker-compose.yml` file. Disabling the Security plugin is not recommended. For more information, see the [Docker release notes](https://github.com/opensearch-project/opensearch-build/tree/main/docker/release#disable-security-plugin-security-dashboards-plugin-security-demo-configurations-and-related-configurations) in GitHub.
```

Comment:
Hi @reify-travis-brown could you rename `Docker release notes` to `Docker image release README`?

Thanks.